engine/userguide/networking/network-debug.md

```diff
@@ -0,0 +1,219 @@
+---
+description: Learn to use the built-in network debugger to debug overlay networking problems
+keywords: network, troubleshooting, debug
+title: Debug overlay or swarm networking issues
+---
+
+Docker CE 17.12 and higher introduce a network debugging tool designed to help
+debug issues with overlay networks and swarm services running on Linux hosts.
+When enabled, a network diagnostic server listens on the specified port and
+provides diagnostic information. The network debugging tool should only be
+started to debug specific issues, and should not be left running all the time.
+
+Information about networks is stored in a database, which can be examined using
```

Comment:
`stored in a database`: how about a little more details around the networkdb:
1. What kind of information is stored?
2. Mention that the database is shared/clustered across all nodes.